extensions/smallrye-stork/runtime/src/main/java/io/quarkus/stork/StorkNetworkInterfaceWrapper.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.stork;
+
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+import java.net.SocketException;
+import java.util.Enumeration;
+
+public class StorkNetworkInterfaceWrapper implements NetworkInterfaceWrapper {
```

Comment:
is it public API?

Reply:
I finally removed it because it's not needed. I must have created it at some point during development to facilitate testing or something like that, I can't remember now.